custom/icds_reports/models/aggregate.py

```diff
@@ -845,6 +844,9 @@ class AggregateComplementaryFeedingForms(models.Model, AggregateMixin):
     class Meta(object):
         db_table = AGG_COMP_FEEDING_TABLE
         unique_together = ('supervisor_id', 'case_id', 'month')  # pkey
+        index_together = [
```

Comment:
are not the indexes  created after inserting the data to have its affect? It looks these indexes will be applied on existing data? m I missing something silly?

Reply:
I don't understand your question. When an index is created it will include all data in that table (unless there's a WHERE condition on the index). As the table is updated with new rows, the index is updated as well

Reply:
When you say `As the table is updated with new rows, the index is updated as well`.  you mean it updates automatically?

Reply:
yes. the database handles that